dts/bindings/i2c/ti,cc32xx-i2c.yaml

```diff
@@ -18,6 +18,12 @@ properties:
       generation: define
       category: required
 
+    "cc32xx,clock-frequency" :
```

Comment:
What about naming it `ip-clock-frequency`? And consistently using `IP_` also in the fixup label? `DT_I2C_0_CLOCK_FREQUENCY` looks like something generated for the `clock-frequency` property.

Reply:
No problem for me. Any other opinions/suggestions?

Reply:
@anangl why `ip-`?

Reply:
the problem with `IP` is that it really reminds of IP block, and this is not the case. So I would instead either:

1. Go with `<name>,clock-frequency`
2. Use something like `<name>,hw-clock-frequency`

@erwango @galak thoughts?

Reply:
We did something on sifive, looks like we used `input-frequency`.  But I'm not keen on propagating that everywhere